src/pages/dashboard/DashboardRow.tsx

```diff
@@ -0,0 +1,302 @@
+  });
+
+  const updateOfferImageMutation = useUpdateOfferImageMutation({
+    onSuccess: async () => {
+      setIsPictureUploadModalVisible(false);
+    },
+  });
+
+  const handleSubmitValid = async ({
+    file,
+    description,
+    copyrightHolder,
+  }: FormData) => {
+    try {
+      setIsImageUploading(true);
+      if (imageToEdit) {
+        await updateOfferImageMutation.mutateAsync({
+          eventId,
+          imageId: imageToEdit.parsedId,
+          description,
+          copyrightHolder,
+          scope,
+        });
+
+        return;
+      }
+
+      await addImageMutation.mutateAsync({
+        description,
+        copyrightHolder,
+        file: file?.[0],
+        language: i18n.language,
+      });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <Inline spacing={5} {...getInlineProps(props)} flex={1}>
+      <Inline width="100" alignItems="center">
+        {imageUrl && (
+          <Image
+            src={imageUrl}
+            alt={title}
+            width={100}
+            height={100}
+            css={`
+              cursor: pointer;
+            `}
+            onClick={() => setIsPictureUploadModalVisible(true)}
+          />
+        )}
+        {!imageUrl && !isFinished && (
+          <span
+            onMouseEnter={() => setIsImageHovered(true)}
+            onMouseLeave={() => setIsImageHovered(false)}
+          >
+            <Box
+              css={`
+                ${!isImageUploading &&
+                `border: 1px solid ${udbMainGrey}; border-radius: 0.5rem;
+              :hover {
+                border: 1px dashed ${udbMainPositiveGreen}; cursor: pointer; 
+              }
+              :active {
+                background-color: ${udbMainLightGreen}; box-shadow: ${getValue(
+                  'boxShadow.small',
+                )};
+              }  
+                `}
+              `}
+              width={100}
+              height={100}
+              display="flex"
+              justifyContent="center"
+              alignItems="center"
+              onClick={() => setIsPictureUploadModalVisible(true)}
+            >
+              {isImageUploading ? (
+                <Spinner />
+              ) : (
+                <ImageIcon
+                  width="50"
+                  color={isImageHovered ? udbMainPositiveGreen : udbMainGrey}
+                />
+              )}
+            </Box>
+          </span>
+        )}
+        {!imageUrl && isFinished && (
+          <Box
+            width={100}
+            height={100}
+            display="flex"
+            justifyContent="center"
+            alignItems="center"
+            backgroundColor={grey3}
+          >
+            <Image
+              src={`/assets/uit-logo.svg`}
+              alt="No image available"
+              width={45}
+              height={45}
+            />
+          </Box>
+        )}
+      </Inline>
+      <PictureUploadModal
```

Comment:
I think the picture upload should probably be on the index page.
Since now there's a modal for every row, which might be a bit overkill.

The modal should be on the index page and you trigger it from the child `DashboardRow`